test/lazy/test_ts_opinfo.py

```diff
@@ -112,7 +109,13 @@ def get_name(op):
         r = op(*args, **kwargs)
         torch._lazy.mark_step()
         torch._lazy.wait_device_ops()
-        prefix = "aten" if op.name in FALLBACK_LIST else "lazy"
+        if op.name in FALLBACK_LIST:
+            prefix = "aten"
+        else:
+            if IS_WINDOWS:
+              prefix = "lazy::torch::lazy::LazyNativeFunctions"
```

Comment:
aha. so basically the code we use to generate the counter names is behaving differently on windows?

Any idea if we could fix that instead? (to make it behave the same as linux) rather than patching the test to expect different behavior on windows?

https://github.com/pytorch/pytorch/blob/master/torch/csrc/lazy/ts_backend/ts_eager_fallback.cpp#L109

Reply:
It's probably related to the name of `struct torch::lazy::LazyTensorNativeFunctions`. For example, this
```
at::Tensor LazyNativeFunctions::alias(const at::Tensor& self) {
  TORCH_LAZY_FN_COUNTER("lazy::");
  return self;
}
```
is expanded to
```
#define TORCH_LAZY_FN_COUNTER(ns) \
  TORCH_LAZY_COUNTER(c10::str(ns, __FUNCTION__), 1)
```
I assume `__FUNCTION__` is mapped to fully-qualified name.

Reply:
Yea, it seems like __FUNCTION__ expands to `alias` on linux but `torch::lazy::LazyTensorNativeFunctions:alias` on windows.  We actually don't want the full name for our purposes (having the counter named `torch::lazy::LazyTensorNativeFunctions:alias` is not ideal).

I wonder if there is some msvc trick or another preprocessor directive we could use so that we can get just the local function name without namespace on both linux/windows?

Reply:
Ideally, we can use `__func__` from C99 standard. However, I found
```
#if defined(_MSC_VER) && _MSC_VER <= 1900
#define __func__ __FUNCTION__
#endif
```
in `pytorch\c10\macros\Macros.h`, which implies visual studio doesn't support it in old versions...